engine/enginepb/hide_sensitive.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package enginepb
+
+import (
+	"github.com/pingcap/tiflow/pkg/logutil"
+	"go.uber.org/zap/zapcore"
+)
+
+var HideSensitiveFunc = logutil.HideSensitiveFunc
```

Comment:
originally I use this fucntion-type variable to break cycle import. But in this PR we can directly use logutil.HideSensitiveFunc in below functions